ideascube/management/commands/tags.py

```diff
@@ -20,6 +21,13 @@ def exit(text, **kwargs):
     sys.exit(1)
 
 
+ponctuation_regex = re.compile(r"([;:.,?!+-@+-/*]+)")
```

Comment:
Not just remove them from the regex.

We do want to weed out tags like `;` after all.

I think a regex is not going to cut it.